### PR TITLE
Changed setup.py to not crash when fim.so download is unavailable for some reason

### PR DESCRIPTION
**Pull request by mdenker**
This should fix assumed install problem in #216

### CODE REVIEW

**Comment by mdenker** [comment]
I am aware that it's not nice to have `except` without an explicit error in a `try` statement, but I am not sure what Error to put -- in doing some testing with things that could go wrong when internet is down or whatever, there were various types of errors that could be raised.

**Comment by mdenker** [comment]
@alperyeg Hi Alper, would you mind having a look at this PR, intended to catch problems during install when, e.g., the network is not available or has a weird proxy or some other reason?

File: setup.py [hunk 1]
```diff
@@ -29,19 +29,20 @@
...
+        arch = '64'
+    else:
+        arch = '32'
+    
+    try:
+        urlretrieve('http://www.borgelt.net/bin' + arch + '/py' + py_ver + '/fim' + oext,
+                    'elephant/spade_src/fim' + oext)
+    except:
+        print("Unable to download fim" + oext + " module.")
+        pass
```
Comment (alperyeg) [inline setup.py]: You don't need `pass`

Comment (mdenker) [reply setup.py]: Ah, no, of course not, thanks...

**Comment by mdenker** [comment]
@alperyeg Thanks for your check.

**Pull request merged**